src/main/osd/osd.c

```diff
@@ -264,6 +264,10 @@ void pgResetFn_osdConfig(osdConfig_t *osdConfig)
 
     osdConfig->osdProfileIndex = 1;
     osdConfig->ahInvert = false;
+
+    for (int i=0; i<OSD_PROFILE_COUNT; i++) {
+        tfp_sprintf(osdConfig->profile[i].name, "OSD_%u", i + 1);
```

Comment:
Rather than pre-filling the profile name with "OSD_#", default to an empty string and do this in the OSD element display. Then it will be consisted with the profile and rateprofile names.

Reply:
That's better, yes.  Changed.